docs/fsharp/style-guide/formatting.md

```diff
@@ -858,90 +657,88 @@ let daysOfWeek' includeWeekend =
 
 In some cases, `do...yield` may aid in readability. These cases, though subjective, should be taken into consideration.
 
-## Formatting if expressions
-
-Indentation of conditionals depends on the size and complexity of the expressions that make them up.
-Write them on one line when:
+## Formatting record expressions
 
-- `cond`, `e1`, and `e2` are short
-- `e1` and `e2` are not `if/then/else` expressions themselves.
+Short records can be written in one line:
 
 ```fsharp
-if cond then e1 else e2
+let point = { X = 1.0; Y = 0.0 }
 ```
 
-If any of the expressions are multi-line or `if/then/else` expressions.
+Records that are longer should use new lines for labels:
 
 ```fsharp
-if cond then
-    e1
-else
-    e2
+// OK
+let rainbow =
+    { Boss = "Jeffrey"
+      Lackeys = ["Zippy"; "George"; "Bungle"] }
 ```
 
-Multiple conditionals with `elif` and `else` are indented at the same scope as the `if` when they follow the rules of the one line `if/then/else` expressions.
+Placing the `{` and `}` on new lines with contents indented is possible, however code formatters may reformat this by default:
 
 ```fsharp
-if cond1 then e1
-elif cond2 then e2
-elif cond3 then e3
-else e4
+// OK, this is the default formatting for code formatters
+let rainbow =
+    { Boss1 = "Jeffrey"
+      Boss2 = "Jeffrey"
+      Boss3 = "Jeffrey"
+      Lackeys = ["Zippy"; "George"; "Bungle"] }
+
+// OK, but code formatters will reformat to the above by default
+let rainbow =
+    {
+        Boss1 = "Jeffrey"
+        Boss2 = "Jeffrey"
+        Boss3 = "Jeffrey"
+        Lackeys = ["Zippy"; "George"; "Bungle"]
+    }
 ```
 
-If any of the conditions or expressions is multi-line, the entire `if/then/else` expression is multi-line:
+The same rules apply for list and array elements.
+
+## Formatting copy-and-update record expressions
+
+A copy-and-update record expression is still a record, so similar guidelines apply.
+
+Short expressions can fit on one line:
 
 ```fsharp
-if cond1 then
-    e1
-elif cond2 then
-    e2
-elif cond3 then
-    e3
-else
-    e4
+// OK
+let point2 = { point with X = 1; Y = 2 }
 ```
 
-If a condition is long, place it on the next line with an extra indent.
-Align the `if` and the `then` keywords.
+Longer expressions should use new lines:
 
 ```fsharp
-if
-    complexExpression a b && env.IsDevelopment()
-    || secondLongerExpression
-        aVeryLongparameterNameOne
-        aVeryLongparameterNameTwo
-        aVeryLongparameterNameThree
-        """
-Multiline
-    string
-        """
-then
-        e1
-    else
-        e2
+// OK
+let rainbow2 =
+    { rainbow with
+        Boss = "Jeffrey"
+        Lackeys = [ "Zippy"; "George"; "Bungle" ] }
```

Comment:
Is there an extra / missing space in these labels?

Reply:
I'm not sure what's suggested here?